packages/electrode-archetype-react-app-dev/config/webpack/util/check-custom-config.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const Fs = require("fs");
+const Path = require("path");
+
+function checkForCustomWebpackConfig(reqConfigFile) {
+  const customFilePath = Path.join(process.cwd(), "webpack.config.js");
+  const defaultFile = reqConfigFile || Path.join("../", "webpack.config.js");
```

Comment:
returning a default relative "../webpack.config.js" that caller would `require` is not deterministic.  what if caller's dir location is different and `require("../webpack.config.js")` is invalid?  Either return absolute path or just throw if caller didn't pass reqConfigFile.

Reply:
modified it. good catch, thanx!